core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/definitions/ldap/implementations/UnboundidLdapService.scala

```diff
@@ -336,6 +339,12 @@ object LdapConnectionConfig {
     final case class CustomUser(dn: Dn, password: PlainTextSecret) extends BindRequestUser
   }
 
+  sealed trait CircuitBreaker
```

Comment:
It looks like we should defined it elsewhere. 

please take a look at:
* `ExternalAuthenticationService`
*  `ExternalAuthorizationService`

It'd be nice to have CB decorator available also for them (the same, like we do in case of caching).

_(We can do it in a second part of the task)_

Reply:
(not resolving it, because in P2 we need to implement the CB for mentioned services)